task/test.js

```diff
@@ -42,9 +45,47 @@ if(!style.metadata['openmaptiles:version']) {
   console.log('WARNING: Style does not contain "openmaptiles:version" metadata.');
 }
 
-var formattedStyleString = mbgl.format(style);
+const formattedStyleString = mbgl.format(style);
 if(styleString.trim() !== formattedStyleString.trim()) {
   console.log(
-    'WARNING: Style is not formatted. You should use "gl-style-format"\n' +
+    'WARNING: Style is not correctly formatted. You should use "gl-style-format"\n' +
     'before commit. See https://github.com/mapbox/mapbox-gl-js/blob/master/src/style-spec/README.md');
 }
+
+
+/* Test i18n config file */
+
+const languageFallbacks = yaml.readSync(path.join(args.style_dir,'i18n.yml')).languageFallbacks
+
+const layersFields = [];
+let error = false;
+
+style.layers.map((layer) => {
+  if(layer['layout'] && layer['layout']['text-field'] && layer['layout']['text-field'] === '{name}') {
```

Comment:
Is that not a bit dangerous to hardcode the value `{name}` here ?  
What if another field is used by default in the future, such as `{name_int}` ?

Reply:
j'ata une suggestion

Reply:
`layer['layout']['text-field'].startsWith('{name');` ?

ok: `{name}`, `{name:fr}`, `{name:en}`, `{name:int}`, `{name:latin} \n {name}`
ko: `{ref}`

Reply:
:+1: I suppose that's a good compromise.